openssl/src/x509/store.rs

```diff
@@ -81,11 +88,53 @@ impl X509StoreBuilder {
 
 impl X509StoreBuilderRef {
     /// Adds a certificate to the certificate store.
-    // FIXME should take an &X509Ref
-    pub fn add_cert(&mut self, cert: X509) -> Result<(), ErrorStack> {
+    pub fn add_cert(&mut self, cert: &X509Ref) -> Result<(), ErrorStack> {
```

Comment:
This is a breaking change. That's why it was a FIXME.

Reply:
I see, would this just be fixed in the next major version?

Reply:
Yes